cachi2/core/models/sbom.py

```diff
@@ -93,3 +94,134 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self) -> "SPDXSbom":
+        """Convert a CycloneDX SBOM to an SPDX SBOM."""
+        return SPDXSbom(
+            packages=[
+                SPDXPackage(
+                    name=component.name,
+                    version=component.version,
+                    package_download_location=component.purl,
```

Comment:
Where is this parameter defined? `package_download_location`

Reply:
I think that this function needs separete and better unittest, if expected is SBOM is really good one, current test testing this is shady as it basically just tests if `f()==f()` with few extra steps